.github/workflows/push-docker.yaml

```diff
@@ -21,6 +27,11 @@ on:
         required: true
         type: string
         description: The repository/name of the docker image to push, e.g., returntocorp/semgrep
+      dry-run:
```

Comment:
This is so ugly. Why do we need to repeat the same thing for workflow_dispatch and worflow_call.
This yaml thing is retarded.

Reply:
I feel this comment falls short of our respect and inclusion values. All technologies have strengths and weaknesses and while I agree that this a weakness, I think it's inappropriate to use `retarded` as a pejorative term here. My understanding from the GitHub documentation on workflow syntax is that the `inputs` context is a child of both `workflow_dispatch` and `workflow_call` and can not be shared between them: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions.

Reply:
Ok sorry, I'll try to better hide my hate for yaml.